config/locales/idv/en.yml

```diff
@@ -56,6 +56,7 @@ en:
       pattern_mismatch:
         ssn: 'Enter a nine-digit Social Security number'
         zipcode: Enter a 5 or 9 digit ZIP Code
+        zipcode_only: Enter a 5 digit ZIP Code
     failure:
```

Comment:
@18F/identity-joy-designers  For this issue, we are limiting the Zip Code input to only 5 digits and will handle 9 digits in [LG-1083](https://cm-jira.usa.gov/browse/LG-10833). I updated the translation files but I don't know if my edits are the correct translation. 

Can you check and/or order a translation for "Enter a 5 digit ZIP Code" ?

![Screenshot 2023-08-24 at 12 54 51 PM](https://github.com/18F/identity-idp/assets/125507397/fe91ab8c-a0c5-4078-a2e6-414d571d16a2)

Reply:
Hi @gina-yamada, these are the translations for that error. I also added them to the [translations doc.](https://docs.google.com/document/d/1vhGmTiF312PjOPWYvJH9oV0X95r7OIgGnQKLDCYGeVI/edit#bookmark=id.n9bqz3cd6v6k)
- Spanish: Ingrese un código postal de 5 dígitos
- French: Entrez un code postal à 5 ​​chiffres